C/common/storage_client.cpp

```diff
@@ -180,6 +258,8 @@ bool StorageClient::readingAppend(const vector<Reading *>& readings)
 		ss << m_pid << "#" << thread_id << "_" << m_seqnum_map[thread_id].load();
 		sto_mtx_client_map.unlock();
 
+		m_logger->info("ss=%s", ss.str().c_str());
```

Comment:
This really does not look like useful info for the user, should be debug at best but probably should be removed.